awx/main/tests/functional/__init__.py

```diff
@@ -0,0 +1,23 @@
+from django.db import connection
+from django.db.models.signals import post_migrate
+from django.apps import apps
+
+
+def app_post_migration(sender, app_config, **kwargs):
```

Comment:
@matburt @AlanCoding @chrismeyersfsu 

This is a somewhat gross hack I had to add to make pytest-django + sqlite happy.  The issue is that for *real* installs and upgrades, we run `awx-manage migrate`, which causes `main_unifiedjob.result_stdout_text` to exist (due to https://github.com/ansible/awx/pull/795/files#r156477183).

Our functional tests work differently.  They set up a sqlite database created by `django-pytest`.  This process explicitly _does not_ run migrations (because it's slow), but instead creates the database schema using the _current state_ of the model definitions.  The result is that the schema spun up by `django-pytest` doesn't have this column (because it no longer exists on `UnifiedJob`).  This migration hook is a bit of a kludge, but it _does_ give us what we want - a final result that looks like the real thing.